src/components/Posts/IndividualPost/IndividualPost.js

```diff
@@ -0,0 +1,101 @@
+import React, { useState, useEffect } from 'react';
+import Typography from '@material-ui/core/Typography';
+import Container from '@material-ui/core/Container';
+import Chip from '@material-ui/core/Chip';
```

Comment:
You should use es6 named imports for material-ui. It helps with refactoring and bundle size. 
[source](https://material-ui.com/guides/minimizing-bundle-size/)